apps/api/src/historical/historical.service.ts

```diff
@@ -131,6 +130,8 @@ export class HistoricalService {
     if (isWeekly) await reset(this.weeklyModel, this.lastWeekModel, doc);
     if (isMonthly) await reset(this.monthlyModel, this.lastMonthModel, doc);
 
+    this.playerService.saveOne(player, false);
```

Comment:
This doesnt fetch from hypixel api.

Reply:
I meant from mongo, shouldn't it just remove them from all historical leaderboards anyways? Also the player object was already saved since `historicalService.resetPlayers` calls `playerService.get` which calls `playerService.saveOne` internally.